lisp/mastodon-toot.el

```diff
@@ -142,14 +141,53 @@ Set `mastodon-toot--content-warning' to nil."
       (mastodon-http--triage response
                              (lambda () (message "Toot toot!"))))))
 
+(defun mastodon-toot--get-acct()
+  "Adds the acct for the handle under the current point to the Kill ring.
+
+This only works if the html received is formatted to contain the domain. 
+it wont work for copying accts from the by-line or if someone entered a 
+handle manually"
+  (interactive)
+  (let ((user-address (plist-get (text-properties-at (point)) 'shr-url)))
+    (if (stringp user-address)
+        (let* ((split-address (split-string-and-unquote user-address "/"))
+               (domain (elt split-address 1))
+               (username (string-remove-prefix "@" (car (last split-address))))
+               (acct (concat "@" username "@" domain)))
+          (kill-new acct)
+          (message (format "Added %s to the kill ring." acct))
+          t)
+      (progn (message (format "No properly formated acct at point:%s." (point)))
+             nil))))
+
+(defun mastodon-toot--process-local (acct)
+  "Adds domain to local ACCT and replaces the curent user name with \"\"."
+  (cond ((string-match-p "@" acct) (concat "@" acct " "))
+        ((string-match-p (mastodon-auth--user-acct) acct) "")
+        (t (concat "@" acct "@"
+                   (cadr(split-string mastodon-instance-url "/" t)) " "))))
+
+(defun mastodon-toot--mentions (status)
+  "Extract mentions from STATUS and process them into a string."
+  (interactive)
+  (let ((mentions (cdr(assoc 'mentions status))))
+    (if (> (length mentions) 0)
+        (mapconcat (lambda(x) (mastodon-toot--process-local
+                               (cdr(assoc 'acct x))))
+                   ;; reverse does not work on vectors in 24.5
+                   (reverse (mapcar (lambda(x) x) mentions)) "") "")))
```

Comment:
Put that third arg to `mapconcat` on its own line to make it clearer.

Also: that should probably be a space `" "` so that names are properly separated and you don't have to add the space in `mastodon-toot--process-local`.

Reply:
I add the spaces in `mastodon-toot-process local` to save us having to remove repeated spaces introduced from the null case (where your username is in the reply list).